connectors/src/connectors/confluence/lib/confluence_client.ts

```diff
@@ -159,6 +162,16 @@ function extractCursorFromLinks(links: { next?: string }): string | null {
   return url.searchParams.get("cursor");
 }
 
+function getRetryAfterDuration(response: Response): number {
+  const defaultValue = 10 * 1000;
+  const retryAfter = response.headers.get("Retry-After"); // https://developer.atlassian.com/cloud/confluence/rate-limiting/
```

Comment:
IIRC headers might be lower cased with `fetch`. Will let you confirm that it works.

Reply:
that's right actually, `Headers`'s constructor lowercases everything, will do the same